specs/global_navigation.adoc

```diff
@@ -0,0 +1,82 @@
+# Initial UI refresh for the grafana integration
+
+Since we are having Grafana with Tendrl, the global menu has to be refined. This spec talks about the modules which we can keep from our previous build (it is available https://github.com/Tendrl/ui/tree/ui-legacy-code[ui-legacy-code] ).
+
+## Problem Description
```

Comment:
Remove the sections `Problem Description` and `Use Cases`, as the introductory paragraph says the same thing.